src/legacy/core_plugins/metrics/public/components/lib/get_interval.js

```diff
@@ -23,6 +23,8 @@ import { relativeOptions } from '../../../../../ui/public/timepicker/relative_op
 
 import { GTE_INTERVAL_RE, INTERVAL_STRING_RE } from '../../../common/interval_regexp';
 
+export const AUTO_INTERVAL = 'auto';
```

Comment:
It would be great to replace all hard coded `auto` interval matches at the `../core_plugins/metrics/**` by this value.

Reply:
We will definitely do it in future. Thanks